addons/actions/src/lib/retrocycle.js

```diff
@@ -2,7 +2,6 @@ import reviver from './reviver';
 import { muteProperty } from './util';
 import { CYCLIC_KEY } from './';
 
-// eslint-disable-next-line no-control-regex
```

Comment:
Looks like it's still needed

Reply:
Yeah, my ide reported it wasn't, will put it back.